imcsdk/apis/admin/ntp.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2016 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This module implements all the ntp related functionality
+"""
+from imcsdk.mometa.comm.CommNtpProvider import CommNtpProvider
+from imcsdk.imccoreutils import _is_valid_arg
+
+
+def _get_ntp_mo(handle=None, from_server=False):
```

Comment:
@waghswapnil I don't think we should mix creating a client copy and server copy here. 
let this code only fetch the server copy. That's what a `get` method should do in this context. the client copy is more of a create and not get. So, whereever you need to get a dn you can use `CommNtpProvider(parent_mo_or_dn="sys/svc-ext").dn` directly